src/main/java/com/practice/designpattern/strategy/TraditionalWay.java

```diff
@@ -0,0 +1,68 @@
+package com.practice.designpattern.strategy;
+
+import java.util.List;
+
+public class TraditionalWay {
+
+    public static void main(String[] args) {
+        List<Integer> nums = List.of(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
+        System.out.println(getContext("Total").executeStrategy(nums));
+        System.out.println(getContext("TotalEven").executeStrategy(nums));
+        System.out.println(getContext("TotalOdd").executeStrategy(nums));
+    }
+
+    private static Context getContext(String action) {
+        Context context = new Context();
+        if (action == "Total") context.setStrategy(new Total());
+        if (action == "TotalEven") context.setStrategy(new EvenTotal());
```

Comment:
Should be if elseif elseif to avoid executing all the branches in sequence all time.

Reply:
if-else added.